api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -804,8 +804,14 @@ def validate_ids_at_provider(cls, ids_at_provider: str) -> list[str] | None:
         return None
 
 
-class GetDatesQueryParams(IndexPaginationQueryParams):
-    pass
+class GetEventStocksQueryParams(IndexPaginationQueryParams):
+    ids_at_provider: str | None = fields.IDS_AT_PROVIDER_FILTER
```

Comment:
```suggestion
    ids_at_provider: list[str] | None = fields.IDS_AT_PROVIDER_FILTER
```
il me semble que c'est une liste

Reply:
Non on attend une string. La liste string c'est ce qui est renvoyé après validation.

Reply:
C'est étrange parce que `ids_at_provider` est déclaré comme étant de type `str | None` (ce qu'on attend effectivement en entrée) alors qu'une fois le modèle construit, on aura `list[str] | None` (construit par le validateur). Non ?

Si c'est bien le cas, est-ce qu'on ne pourrait pas plutôt déclarer ces `ids` comme une liste et avoir un validateur avec l'option `pre=True` ?

Je comprends que les autres modèles fonctionnent de cette manière, mais j'ai peur que ce soit trompeur (et dans ce cas, autant commencer à corriger le soucis tout de suite).

Qu'en dis-tu ?

Reply:
Non mais si on fait ça `ids_at_provider` apparaîtra comme une liste dans la doc et on ne veut pas.